build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -391,9 +391,9 @@ CONFIG_ARCH_USE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="2048,512,1024,1024,1024,1024,1024,128,128,"
-CONFIG_FLASH_PART_TYPE="kernel,none,bin,bin,bin,bin,smartfs,ftl,config,"
-CONFIG_FLASH_PART_NAME="kernel,app,micom,micom,wifi,wifi,userfs,ftl,config,"
+CONFIG_FLASH_PART_SIZE="2048,512,3072,128,128,"
```

Comment:
Wasted 2048 blocks, you have no plan??
Is binary located under the mnt folder?
I mean binary & user file located same folder? Then If user write file many times, then any block of your binary can be relocated to another block because of wear-level.
To prevent this, you should use 2 mount points.